src/bio/bio_xstream.c

```diff
@@ -1116,14 +1112,15 @@ bio_nvme_configured(enum smd_dev_type type)
 
 	if (type >= SMD_DEV_TYPE_MAX)
 		return true;
+
 	/*
-	 * FIXME When storage class 'ram' is used in server yaml, control
-	 * plane regards MD on SSD is enabled and meta role will be set to
-	 * bdev name, that'll fail CI tests before the md on SSD feature is
-	 * fully implemented.
+	 * As wal and meta roles are not supported on the same device, single NVMe tier is not a
+	 * supported configuration for MD on SSD.
```

Comment:
WAL and meta on same device is supported, and why we want to introduce the restriction (single NVMe tier isn't supported for md-on-ssd)?

Reply:
I understood from @wangshilong that WAL+meta is not currently supported https://daosio.atlassian.net/browse/DAOS-12449?focusedCommentId=109897 but if we don't want to introduce this restriction then I need to apply the role assignments explicitly in CI to provide a path to run tmpfs+single bdev tier in non-MD-on-SSD mode. This can be done.

Reply:
Sorry for misunderstaing, i thought you mean WAL+Meta query is unsupported.